CONTRIBUTING.md

```diff
@@ -2,13 +2,16 @@
 
 When contributing to this repository, please first discuss the change you wish to make via an issue before making a change. Feel free to create a pull request to [resolve an issue](https://github.com/jackdomleo7/Checka11y.css/issues), with a solution to an item on our [project board](https://github.com/jackdomleo7/Checka11y.css/projects/1) or for any of your own ideas. Please provide as much detail and as many resources that will backup why you made the change.
 
+When looking to contribute, please comment on issues asking if you can do that work, and only look at doing work that is **not labelled** as `question` or `needs more info` because these issues require more research and discussion before doing. Please also do not ask to do work that is already assigned to someone (you can see who is assigned to an issue by looking at the "assignee").
+
```

Comment:
@jackdomleo7 Just wanted to note that should we be asking people to ask us to assign tasks? I'm far more in favour of a "pick up anything which isn't already assigned" approach. This potentially adds a lot of friction before people can get started. I'd rather just allow them to jump straight in assuming it's ready to work on and not being stolen from somebody else who already claimed it.

Reply:
We'll wait for @devindford's response, but apparently, someone told me they can't assign themselves? 🤔

Reply:
I'll merge it for now, and we can review it some more. 😊

Reply:
Hmm yep seems like self-assignment is a risk factor (as it also opens up the ability for assignment to other users too) which is why it's not been implemented. 

In that case, requests are fine!